packages/xrpl/test/models/accountSet.test.ts

```diff
@@ -10,7 +10,7 @@ import { validateAccountSet } from 'xrpl-local/models/transactions/accountSet'
 describe('AccountSet', function () {
   let account
 
-  beforeEach(function () {
+  beforeEach(() => {
```

Comment:
Can these changes also be undone? It just inflates the number of files affected, and seems easy enough to undo with a ctrl-f-replace.

Reply:
Fixed